### ISSUE TITLE
[Bug][Worker] Task log full

### ISSUE THREAD

**Issue by SbloodyS** [apache/dolphinscheduler#6000]
shell task error
```
java.lang.IllegalStateException: Queue full
	at java.util.AbstractQueue.add(AbstractQueue.java:98)
	at org.apache.dolphinscheduler.server.worker.task.AbstractCommandExecutor.lambda$parseProcessOutput$0(AbstractCommandExecutor.java:356)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```
**Which version of Dolphin Scheduler:**
 -[1.3.7]

**Comment by SbloodyS** [comment]
this scripts works fine in 1.3.6

**Comment by SbloodyS** [comment]
https://github.com/apache/dolphinscheduler/blob/f013ef832aee6d0513ddc2af8cd41d4c16ff0ac9/dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/Constants.java#L394

it seems DEFAULT_LOG_BUFFER_SIZE too small

**Comment by ruanwenjun** [comment]
@SbloodyS Sorry, this is a bug, there are two solutions to solve
1. You can edit the AbstractCommandExecutor line 102
```java
this.logBuffer = new LinkedBlockingQueue<>(Constants.DEFAULT_LOG_BUFFER_SIZE);
```
to 
```java
this.logBuffer = new LinkedBlockingQueue<>();
```

2. You can modify all `logBuffer.add()` in `AbstractCommandExecutor` and use `logBuffer.put()`

**Comment by SbloodyS** [comment]
thanks.

**Comment by CalvinKirs** [comment]
deeply thanks for your feedback, I will fix it to 1.3.8 release and publish it.

**Comment by CalvinKirs** [comment]
fix #6005
